ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1038,6 +1038,12 @@ ethernet_interfaces:
     traffic_policy:
       input: < ingress traffic policy >
       output: < egress traffic policy >
+    sflow:
+      enabled: < true | false >
```

Comment:
we should use "enable" as it is the variable name in EOS.

Reply:
made requested change

```yaml
    sflow:
      enable: < true | false >
      egress:
        # Support for either "enable" or "unmodified_enable" is platform dependent.
        enable: < true | false >
        unmodified_enable: < true | false >
```